### ISSUE TITLE
Community QA: testing Atlas for YPP Release

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/pioneer#4245]
@dmtrjsg commented on [Thu Mar 16 2023](https://github.com/Joystream/atlas/issues/3944)

## Context

⚠️ I am raising this issue in Pioneer Repo since suggested stakeholders already have access and permissions to manipulate issues there, but the Issue pertains ATLAS product. 

We would greatly benefit from having a more diverse set of users to try and test the YPP flow on the dev.gleev.xyz before the release to production and exposing the service to YT creators. 

Some bugs are popping up, especially when user tries to deviate from the "happy path". And at this point it seems like best strategy is to involve someone into testing who was not closely working with the design and dev team. to "stress-test" the solution. 

So on that basis I suggest to get Ivan_T and Oig_res, QA engineers from Builders WG to dedicate some time on testing the YPP Feature.

# Scope
- [ ] Test the sign up and videos sync for YPP project. Identify possible bugs, points of confusion and report your feedback in the comments to this issue. 

Basically we need you to try and break it  :) as happy path was tested quite a few times successfully.

Test Env: dev.gleev.xyz

Default env is set up to `Atlas-dev`, which is the right one. Just in case: to change networks press ctrl+shift+d and there will be a popup with network selection.

## ℹ️  Test Data
You would need to pre-prepare test data to be able to sign up (there are some test conf requirements for YT channels who can participate)

To sign up successfully: 
1. create fresh google account (mail address)
2. using this address go and create new YT channel 
3. Share your channel url so I can subscribe to it (it needs at least 1 subscriber to qualify)
4. upload at least 2 videos 
5. Wait for 1hr

**Comment by dmtrjsg** [comment]
@ivanturlakov  thank you for the quick turnaround ⭐ 

I suspect that may be due to some of the criteria for YT channel been unmet:
- at least 1 subscriber to the channel
- at least 2 videos
- at least 1 hr old 

In order to be sure in the console error open the details (chevron looking >)
Then go "response" -> "data" and see what the error code is. 

The error codes PR is getting merged tomorrow, which would surface the error to the user in the toaster notiff, so possibly worth to retest after its merged if we are unable to identify the root cause from console.
- https://github.com/Joystream/atlas/pull/3924

**Comment by ivanturlakov** [comment]
@dmtrjsg criteria for YT channel are ok,

<img width="985" alt="Снимок экрана 2023-03-16 в 22 21 15" src="https://user-images.githubusercontent.com/46903215/225744603-38d2e932-2ce7-43ec-8638-6ea041cfc2fc.png">

I added a referrerChannel and was able to proceed, now the YPP page looks like this

<img width="1344" alt="Снимок экрана 2023-03-16 в 22 30 13" src="https://user-images.githubusercontent.com/46903215/225746001-caf09419-3509-4b01-b8a3-026034b77cb9.png">

**Comment by dmtrjsg** [comment]
Thank you @ivanturlakov, the issue you reported was already been fixed at the time of testing but we have not had a chance to merge it just yet: 
- https://github.com/Joystream/atlas/issues/3943

**Comment by ivanturlakov** [comment]
Yesterday the initial sync was successful, and this morning all videos were available on the Gleev. Also today I added a new video to YouTube, and after a while, it appeared on my channel in Gleeve.

If there is time for this, it would be nice to show an indication that the synchronization process is active, 3 out of 10 videos are loaded, or something like that

<img width="1440" alt="Снимок экрана 2023-03-17 в 15 19 30" src="https://user-images.githubusercontent.com/46903215/225920372-91e33144-1821-4676-b5f5-cb4498501c7a.png">

**Comment by 2333-Hanzo** [comment]
YT link- https://youtube.com/@HanzoHazashi241?feature=shares


The sync had issues in the first 2 tries and there was a runtime error but on the third try it was successful but Synced videos are yet to show on Gleev.

**Comment by Counsellor-sama** [comment]
YT videos now appear on my Gleev channel https://dev.gleev.xyz/studio/videos

**Comment by Codefikeyz** [comment]
Fresh google account (mail address): 01testcode@gmail.com
YT Channel name: Testcode01 @testcode01
YT Channel URL: https://www.youtube.com/channel/UCV0rRcBagmdVGM2VfeN4WaQ
Two videos were initially uploaded on 20th March 2023
Tried syncing on 21st March 2023, for sometime nothing worked.
Trying to seek solution, I thought the funny video uploaded with the other video was the issue for the failed sync, as it was not crypto related.
I deleted the short funny video on YT and uploaded two short videos
Coming back to gleev the old videos were uploaded, 1 was successful while the other wasn't, as seen below
![Screenshot_10](https://user-images.githubusercontent.com/123887455/226673294-71a750ed-140c-4620-a097-a4a49ed7b721.png)
This went on for over 3 hours while the other two new videos uploaded on YT appeared on gleev without any issue
On going back to gleev after sometime, I discovered the video file having issues uploading was actually a duplicate of the first video, and has successfully been uploaded. See image below
![upload complete](https://user-images.githubusercontent.com/123887455/226675325-ddb2823a-f02d-4942-b18c-f1d8d2860787.png)
Also below is a screenshot of the 3 videos on my channel 
![YT Videos](https://user-images.githubusercontent.com/123887455/226675600-7dfdf05a-1ed9-4158-8d13-d8ee3d78e9a2.png)

Feedback - I think the issues encountered was 
1. a failed sync process in the first time with the funny video I later deleted thinking that was the issue
2. Failure to do a fresh sync at a new try of file delete + 2 files upload, instead of picking up with the initial sync command or whatever the reason might be for gleev uploading duplicate content, maybe cos of failed sync - file delete on YT - upload two new files - Sync auto resumes - upload of duplicate files.
